apps/api_banking_web/lib/api_banking_web/controllers/account_controller.ex

```diff
@@ -0,0 +1,24 @@
+defmodule ApiBankingWeb.AccountController do
+  use ApiBankingWeb, :controller
+
+  def get_statement(conn, %{"account_id" => acc_id}) do
+    with {:uuid, {:ok, _}} <- {:uuid, Ecto.UUID.cast(acc_id)},
+         {:ok, statement} <- ApiBanking.get_statement(acc_id) do
+      send_json(statement, conn, 200)
+    else
+      {:uuid, :error} ->
+        %{type: "bad_request", description: "Account id must be a valid UUID"}
+        |> send_json(conn, 400)
```

Comment:
Pipe de uma linha só eu acho que o credo reclama também:
```suggestion
        send_json(%{type: "bad_request", description: "Account id must be a valid UUID"}, conn, 400)
```